commands/blacklist/viewbl.js

```diff
@@ -0,0 +1,16 @@
+const { SlashCommandBuilder } = require('discord.js');
```

Comment:
Sometimes you have single quotes, sometimes you have double-quotes. Please keep it consistent.